contracts/plugins/trading/DutchTrade.sol

```diff
@@ -73,6 +88,7 @@ contract DutchTrade is ITrade {
     using SafeERC20 for IERC20Metadata;
 
     TradeKind public constant KIND = TradeKind.DUTCH_AUCTION;
+    BidType public bidType = BidType.NONE;
```

Comment:
This line is slightly weird because the contract is used as a clone. It's fine, because `bidType` will be 0 in the proxies on deployment, but having the assignment at all feels misleading. I'd rather leave the assignment out and throw a comment in there `// = BidType.NONE`. Bit of a nit I know